src/Symfony/Component/Mime/Crypto/SMimeEncrypter.php

```diff
@@ -24,17 +24,21 @@ final class SMimeEncrypter extends SMime
 
     /**
      * @param string|string[] $certificate The path (or array of paths) of the file(s) containing the X.509 certificate(s)
-     * @param int             $cipher      A set of algorithms used to encrypt the message. Must be one of these PHP constants: https://www.php.net/manual/en/openssl.ciphers.php
+     * @param int|null        $cipher      A set of algorithms used to encrypt the message. Must be one of these PHP constants: https://www.php.net/manual/en/openssl.ciphers.php
```

Comment:
I'm not questioning this change ... but for my own information, I'd like to know why changing this to `int|null`. Both solutions seem to be equivalent, but to me, not allowing `null` is "better". Thanks!

Reply:
The answer is on the line below: because the constant may be undefined, so it cannot be used ad default.

Reply:
Ahhhh It was so obvious that I didn't see it 🤦‍♂  Sorry!